superset/initialization/__init__.py

```diff
@@ -221,6 +222,7 @@ def init_views(self) -> None:
         appbuilder.add_api(QueryRestApi)
         appbuilder.add_api(ReportScheduleRestApi)
         appbuilder.add_api(ReportExecutionLogRestApi)
+        appbuilder.add_api(ResetRestApi)
```

Comment:
Can we put this add behind a boolean config that defaults to `False`? Although useful for local dev work, this feels a little dangerous to have enabled by default

Reply:
Sure, this is a good idea and will add it behind a feature flag, where the **default behavior** of the feature flag would be to disable this API (by default). That way, no one can run it against PROD, unless enabled.

CC: @john-bodley , @mistercrunch , @craig-rueda , @michael-s-molina